src/SimpleSimulator.cpp

```diff
@@ -98,6 +99,8 @@ po::variables_map process_program_options(int argc, char** argv) {
         ("output-file,o", po::value<std::string>()->value_name("<out file>")->required(), "path for the CSV file containing output information about the jobs in the simulation")
 
         ("xrd-blocksize,x", po::value<double>()->default_value(xrd_block_size), "size of the blocks XRootD uses for data streaming")
+
+        ("cache-scope", po::value<std::string>()->default_value("local"), "Set the network scope in which caches can be found:\n local: only caches on same machine\n network: caches in same network zone\n recnetwork: also include caches in subzones")
```

Comment:
You could introduce a check that the choices for the zone are correct.

Reply:
Done